lib/simctl.js

```diff
@@ -6,8 +6,7 @@ import _ from 'lodash';
 
 const log = getLogger('simctl');
 
-async function simExec (command:string, timeout:number, args:Array = [],
-    env = {}) {
+async function simCommand (command:string, timeout:number, args:Array = [], env = {}, executingFunction) {
```

Comment:
yeah works great.
I see what you mean though, I'll default it to simExec, how about that?